pom.xml

```diff
@@ -122,7 +122,7 @@
 			</snapshots>
 			<id>proto-staging</id>
 			<name>0.24.x Protobufs</name>
-			<url>https://oss.sonatype.org/content/repositories/comhederahashgraph-1410</url>
+			<url>https://oss.sonatype.org/content/repositories/comhederahashgraph-1418</url>
```

Comment:
This file will need to be adjusted to production versions prior to merge.

Reply:
We have tended to accept `-alpha.x` proto artifacts in Services `master` b/c it's so difficult to truly front-load all the proto changes targeted for a release. 😞